src/pages/settings/ExitSurvey/ExitSurveyResponsePage.tsx

```diff
@@ -43,8 +46,19 @@ function ExitSurveyResponsePage({draftResponse, route, navigation}: ExitSurveyRe
     const StyleUtils = useStyleUtils();
     const {keyboardHeight} = useKeyboardState();
     const {windowHeight} = useWindowDimensions();
-    const {top: safeAreaInsetsTop} = useSafeAreaInsets();
     const {inputCallbackRef, inputRef} = useAutoFocusInput();
+    const [headerTitleHeight, setHeaderTitleHeight] = useState(0);
+
+    // Device safe area top and bottom insets.
+    // When the keyboard is shown, the bottom inset doesn't affect the height, so we take it out from the calculation.
+    const {top: safeAreaInsetsTop, bottom: safeAreaInsetsBottom} = useSafeAreaInsets();
+    const safeAreaInsetsBottomValue = !keyboardHeight ? safeAreaInsetsBottom : 0;
+    // FormWrapper bottom padding
+    const {paddingBottom: formPaddingBottom} = useStyledSafeAreaInsets();
+    const formPaddingBottomValue = formPaddingBottom || styles.pb5.paddingBottom;
+    // Extra bottom padding in FormAlertWithSubmitButton
+    const safePaddingBottomStyle = useSafePaddingBottomStyle();
+    const safePaddingBottomStyleValue = 'paddingBottom' in safePaddingBottomStyle ? (safePaddingBottomStyle.paddingBottom as number) : 0;
```

Comment:
For the recent fix, I add another inset from `useStyledSafeAreaInsets` to use in the calculation.

The form currently has 3 bottom spacings. 1 from safe area insets (ScreenWrapper), 1 from styled safe area insets,
https://github.com/Expensify/App/blob/117b96112b4b2de8ead192c9c1da619edb9860fa/src/components/Form/FormWrapper.tsx#L107-L110

and the last 1 is from `useSafePaddingBottomStyle`.
https://github.com/Expensify/App/blob/117b96112b4b2de8ead192c9c1da619edb9860fa/src/components/FormAlertWithSubmitButton.tsx#L99-L100

`useSafeAreaInsets` is added in ScreenWrapper so it won't overlap with this.
<img width="424" alt="image" src="https://github.com/user-attachments/assets/9d77add1-0da9-4cb1-a6a6-bcd804701a2b">

The other 2 just add additional spacing in 2 different places and 2 different values.

FormAlertWithSubmitButton which uses `useSafePaddingBottomStyle` adds 20px while FormWrapper uses `useStyledSafeAreaInsets` with the same value as `useSafeAreaInsets` but multiplied by 0.7 and fallback to 20px if the bottom inset is not available (for Android).
https://github.com/Expensify/App/blob/117b96112b4b2de8ead192c9c1da619edb9860fa/src/styles/utils/index.ts#L329-L332